Gopkg.toml

```diff
@@ -87,3 +87,7 @@
 [[constraint]]
   name = "k8s.io/kubernetes"
   version = "1.9.1"
+
+[[constraint]]
```

Comment:
Just want to point out that I did not fix the problem that I had with vendor:

```
$ make vendor
Solving failure: No versions of github.com/automationbroker/broker-client-go met constraints:
        master: Could not introduce github.com/automationbroker/broker-client-go@master, as it has a dependency on k8s.io/client-go with constraint release-6.0, which has no overlap with existing constraint kubernetes-1.9.1 from (root)
        master: Could not introduce github.com/automationbroker/broker-client-go@master, as it has a dependency on k8s.io/client-go with constraint release-6.0, which has no overlap with existing constraint kubernetes-1.9.1 from (root)
        v1.0.0: Could not introduce github.com/automationbroker/broker-client-go@v1.0.0, as it has a dependency on k8s.io/client-go with constraint release-6.0, which has no overlap with existing constraint kubernetes-1.9.1 from (root)
make: *** [Makefile:18: vendor] Error 1
```
I just worked around it since it wasn't obvious to me the correct way to fix it.